library/NotificationCenter/ContaoHelper.php

```diff
@@ -66,6 +66,54 @@ public function sendRegistrationEmail($intId, $arrData, &$objModule)
         }
     }
 
+    /**
+     * Send a activation e-mail
+     * @param object
+     * @param object
+     */
+    public function sendActivationEmail($objMember, &$objModule)
+    {
+        if (!$objModule->nc_notification) {
+            return;
+        }
+
+        $arrTokens = array();
+        $arrTokens['admin_email'] = $GLOBALS['TL_ADMIN_EMAIL'];
+        $arrTokens['domain'] = \Environment::get('host');
+
+        // Support newsletters
+        if (in_array('newsletter', $this->Config->getActiveModules()))
```

Comment:
A lot of this code is duplicated from `sendRegistrationEmail()`. That's not really what we want, right? ;)